docs/usage/self-hosted-experimental.md

```diff
@@ -100,6 +100,14 @@ Allowed values:
 
 Default value: `asc`.
 
+## `RENOVATE_X_REBASE_PAGINATION_LINKS`
+
+<!-- prettier-ignore -->
+!!! note
+  For the GitHub Enterprise Server platform only.
```

Comment:
Wrong, remove

Reply:
Should I phrase it as being for GitHub only (not just enterprise) for now since the flag is read in github.ts?  Or just remove the note?

Reply:
no, as this class is also used to lookup github deps when renovate is running on other platforms

Reply:
Removed in https://github.com/renovatebot/renovate/pull/19888/commits/b0086878e1481eb58c4c5706a77aa1f84e51ca75